packages/client/modules/userDashboard/components/OrgBilling/BillingForm.tsx

```diff
@@ -135,17 +125,11 @@ const BillingForm = (props: Props) => {
         return
       }
       const {error} = await stripe.confirmCardPayment(stripeSubscriptionClientSecret)
-      setIsLoading(false)
       if (error) {
         setErrorMsg(error.message)
+        setIsLoading(false)
```

Comment:
The component changes if the user successfully upgrades, so we only need to set `isLoading` to false if it fails